packages/theme/css/characteristics/selectable.css

```diff
@@ -15,45 +15,54 @@
   --salt-selectable-borderColor-selectedDisabled: var(--salt-palette-interact-border-activeDisabled);
   --salt-selectable-borderColor-disabled: var(--salt-palette-interact-border-disabled);
 
-  --salt-selectable-foreground-partial: var(--salt-palette-interact-foreground-partial);
-  --salt-selectable-foreground-partialDisabled: var(--salt-palette-interact-foreground-partialDisabled);
+  --salt-selectable-foreground: var(--salt-palette-interact-foreground);
+  --salt-selectable-foreground-disabled: var(--salt-palette-interact-foreground-disabled);
+  --salt-selectable-foreground-hover: var(--salt-palette-interact-foreground-hover);
+  --salt-selectable-foreground-selected: var(--salt-palette-interact-foreground-active);
```

Comment:
would there be a `--salt-selectable-foreground-selected-hover` also? I believe it needs to be created in figma also